core/server/data/migrations/versions/3.2/01-add-cancel-at-period-end-to-subscriptions.js

```diff
@@ -0,0 +1,25 @@
+const commands = require('../../../schema').commands;
```

Comment:
@allouis & @rishabhgrg this commit has a tiny migration and some schema changes. Would need your :+1: for this as usual :wink:

Reply:
@gargol Looks good 👍